lib/Core/Executor.cpp

```diff
@@ -93,7 +93,7 @@ using namespace llvm;
 using namespace klee;
 
 
-namespace {
+namespace klee {
```

Comment:
Why do you move the options from the anonymous name space?

Reply:
Because I need to use in other files the categories defined there.  I guess I could move only the categories to the klee namespace (and later move only the options that might be needed in other files too).  Does this sound better?